src/components/interactive-map.js

```diff
@@ -24,11 +25,11 @@ const propTypes = Object.assign({}, StaticMap.propTypes, {
   minPitch: PropTypes.number,
 
   /**
-   * `onChangeViewport` callback is fired when the user interacted with the
+   * `onViewportChange` callback is fired when the user interacted with the
```

Comment:
speaking as someone who admittedly hasn't used `react-map-gl` much but _has_ used other mapping frameworks, `onViewportChange` or `onViewportChanged` should work. Other frameworks generally provide a higher level of granularity, like `onZoom`, `onPan`, but I think `viewport` as one overall concept works.